connman/src/service.c

```diff
@@ -5996,6 +6026,10 @@ static void service_append_added_foreach(gpointer data, gpointer user_data)
 
 		append_struct(service, iter);
 		g_hash_table_remove(services_notify->add, service->path);
+	} else if (g_hash_table_lookup(services_notify->update, service->path)) {
+		DBG("updated %s", service->path);
+
+		append_struct_service(iter, append_dict_properties_updated, service);
```

Comment:
I guess here this could follow the same approach as adding, so `g_hash_table_remove(services_notify->update, service->path);` could be here for consistency?

Reply:
Thanks a lot for thre review, I have fixed that test that, everything works fine